testSrc/unit/io/flutter/run/SdkFieldsTest.java

```diff
@@ -21,18 +21,22 @@
   public void shouldReadFieldsFromXml() {
     final Element elt = new Element("test");
     addOption(elt, "filePath", "lib/main.dart");
+    addOption(elt, "programArgs", "--debug");
```

Comment:
Good question.  In general, I'd like to add some validation but haven't thought to hard about it.  I'll add a TODO.